mapswipe_workers/mapswipe_workers/firebase_to_postgres/archive_project.py

```diff
@@ -15,24 +15,54 @@ def archive_project(project_ids: list) -> None:
     """
     for project_id in project_ids:
         logger.info("Archive project with the id {0}".format(project_id))
-        logger.info(
-            "Delete results, groups and tasks of project with the id {0}".format(
-                project_id
-            )
-        )
+        logger.info("Delete results of project with the id {0}".format(project_id))
```

Comment:
Why not use f-strings? If you're using python 3.6+ everywhere, they'd be easier to read. This line would be:
`logger.info(f"Delete results of project with the id {project_id}")`.
Have a look at https://realpython.com/python-f-strings/ for more explanations.

Reply:
yep, we have been using this before as well. Don't know we didn't do it here already. Adjusted